triage/forms.py

```diff
@@ -136,6 +145,26 @@ class CompaniesHouseSearchForm(forms.Form):
     term = forms.CharField()
 
 
+class SectorForm(forms.Form):
+    sector = forms.ChoiceField(
```

Comment:
you can now use export-components's `fields.ChoiceField` - no need to set `label_suffix` and adds some style classes for us